the_debuginator.h

```diff
@@ -1552,6 +1581,8 @@ void debuginator_update_filter(TheDebuginator* debuginator, const char* wanted_f
 		fallback->parent->folder.hot_child = fallback;
 	}
 
+	debuginator__set_item_total_height_recursively(debuginator->root, debuginator->item_height);
```

Comment:
It should be enough to call it once on the root _after_ we're done calculating filtering status, but it needs to be done to account for changes to children before you in the tree.